### ISSUE TITLE
Can we create a workflow using this package?

### ISSUE THREAD

**Issue by ghost** [common-workflow-language/python-cwlgen#11]
Tried creating a workflow using this pacakge just like 'CommandLineTool' creation example. But looks like this tool does not support the workflow creation, got an error while doing so. Is this correct understanding?

**Comment by illusional** [comment]
Hey _ghost_, it will probably be released in v0.3.0 (it's in this repository now). You could then use classes to generate Workflows, as well as CommandLineTools. You could do the following:

```python
import cwlgen

w = cwlgen.Workflow(workflow_id=None, label=None, doc=None, cwl_version='v1.0')
# and then set inputs
w.inputs.append(cwlgen.InputParameter(param_id, label=None, secondary_files=None, param_format=None, streamable=None, doc=None, input_binding=None, default=None, param_type=None)

w.export() # writes CWL to console
```

Not that this doesn't logically check your workflow, we'd recommend something like [CWLTool](https://github.com/common-workflow-language/cwltool/) for that.

**Comment by illusional** [comment]
Hi again _ghost_,
[v0.3.0](https://github.com/common-workflow-language/python-cwlgen/tree/v0.3.0-RC) has been released so I'm going to close this issue. The documentation has been updated on the README and in the [documentation](https://python-cwlgen.readthedocs.io/en/latest/classes.html)
